Command/DeleteOrphanIndexesCommand.php

```diff
@@ -36,19 +36,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->aliasService->build();
         foreach ($this->aliasService->getOrphanIndexes() as $index) {
-            $this->deleteOrphanIndex($index, $output);
+            $this->deleteOrphanIndex($index);
         }
     }
 
-    private function deleteOrphanIndex($index, OutputInterface $output)
```

Comment:
Type hint for the funciton parameters is missing

Reply:
Also fixed